modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,40 +10,111 @@
 
 |===
-|**Name** |**Default** |**Description**
-|`ApplicationInstanceLabelKey` |`mycompany.com/appname`|The `metadata.label` key name where Argo CD injects the app name as a tracking label.
-|`ApplicationSet` |__<Object>__|`ApplicationSet` controller configuration options.
-|`ConfigManagementPlugins`    |__<empty>__|Add a configuration management plugin.
-|`Controller`    |__<Object>__|Argo CD Application Controller options.
-|`Dex`    |__<Object>__|Dex configuration options.
-|`Disable Admin`    |`false` |Disable the admin user.
-|`GATrackingID`    |__<empty>__|Use a Google Analytics tracking ID.
-|`GAAnonymizeusers`    |`false` |Enable hashed usernames sent to google analytics.
-|`Grafana`    |__<Object>__|Grafana configurable options.
-|`HA`    |__<Object>__|High availablity options.
-|`HelpChatURL`    |`https://mycorp.slack.com/argo-cd` |URL for getting chat help (this will typically be your Slack channel for support).
-|`HelpChatText`    |`Chat now!`|The text appears in a text box for getting chat help.
-|`Image`    |`argoproj/argocd` |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
-|`Import`    |__<Object>__|Import configuration options.
-|`Ingress`    |__<Object>__|Ingress configuration options.
-|`InitialRepositories`    |__<empty>__|Initial Git repositories to configure Argo CD to use upon creation of the cluster.
-|`RepositoryCredentials`    |__<Object>__|Git repository credential templates to configure Argo CD to use upon creation of the cluster.
-|`InitialSSHKnownHosts`    |__<default_Argo_CD_Known_Hosts>__|Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.
-|`KustomizeBuildOptions`    |__<empty>__|The build options and parameters to use with `kustomize build`.
-|`OIDCConfig` |__<empty>__|The OIDC configuration as an alternative to Dex.
-|`NodePlacement` |__<empty>__|Add the `nodeSelector` and the `tolerations`.
-|`Prometheus` |__<Object>__|Prometheus configuration options.
-|`RBAC` |__<Object>__|RBAC configuration options.
-|`Redis` |__<Object>__|Redis configuration options.
-|`ResourceCustomizations` |__<empty>__|Customize resource behavior.
-|`ResourceExclusions` |__<Object>__|Completely ignore entire classes of resource group.
-|`ResourceInclusions` |__<Object>__|The configuration to configure which resource group/kinds are applied.
-|`Server` |__<Object>__|Argo CD Server configuration options.
-|`SSO` |__<Object>__|Single sign-on options.
-|`StatusBadgeEnabled` |`true` |Enable application status badge.
-|`TLS` |__<Object>__|TLS configuration options.
-|`UserAnonyousEnabled` |`true` |Enable anonymous user access.
-|`Version` |`v2.2.2` |The tag to use with the container image for all Argo CD components.
+|**Name** |**Description** |**Default** | **Properties**
+|`ApplicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`mycompany.com/appname`|
```

Comment:
The default for this is `app.kubernetes.io/instance` - it is already wrong in the upstream's Operator documentation.